package.json

```
@@ -113,7 +113,7 @@
   },
   "homepage": "https://renovatebot.com",
   "engines": {
-    "node": "^12.13.0 || >=14.15.0",
+    "node": "^12.17.0 || >=14.15.0",
```

```suggestion
    "node": "^12.22.0 || >=14.15.0",
```
@JamieMagee @rarkins Should we raise to have full esm support? 
https://nodejs.org/en/blog/release/v12.22.0/
https://nodejs.org/docs/latest-v12.x/api/esm.html#esm_modules_ecmascript_modules